### PR TITLE
Added FileName() to generate random filenames

### PR DESCRIPTION
**Pull request by raff**
liberaly inspired by "https://www.npmjs.com/package/natural-filename-generator"

FileName(ext) generates a random multi-world filename with the provided extension.

### CODE REVIEW

File: random_data.go [hunk 1]
```diff
@@ -362,3 +385,75 @@ func FullDateInRange(dateRange ...string) string {
 func Timezone() string {
 	return randomFrom(jsonData.Timezones)
 }
+
+///////////// random filenames ///////////////////////////
```
Comment (Pallinder) [inline random_data.go]: We don't really use this commenting style anywhere else, perhaps we should remove it here as well?

File: random_data.go [hunk 2]
```diff
@@ -362,3 +385,75 @@ func FullDateInRange(dateRange ...string) string {
...
+
+type generator func() string
+
+var (
+	generators = []generator{
+		Topic, // a topic
```
Comment (Pallinder) [inline random_data.go]: No need for these comments, code explains that it's a topic.

**Comment by raff** [comment]
I removed the comments. For the changes to fullprofile.go and fullprofile_test.go, actually the only changes compared to the original version is that I did run "go fmt". If you want I can revert those changes.

**Comment by Pallinder** [comment]
There also seems to be a failing test.

**Comment by Pallinder** [comment]
Closing this one since it's stale.

**Pull request closed without merging**